src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs

```diff
@@ -1,12 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace SFA.DAS.RoatpOversight.Domain
+﻿namespace SFA.DAS.RoatpOversight.Domain
 {
-    public static class GatewayReviewStatus
+    public class GatewayReviewStatus
```

Comment:
Should be a static class as it contains only contants

Reply:
Feels like this class is lacking some of the other statuses. For example
 
"In Progress"
"Clarification sent"